pkg/gatewayserver/io/basicstationlns/basicstationlns.go

```diff
@@ -395,6 +395,7 @@ func (s *srv) handleTraffic(c echo.Context) (err error) {
 					"station":  version.Station,
 					"firmware": version.Firmware,
 					"package":  version.Package,
+					"platform": fmt.Sprintf("%s - Firmware %s - Protocol %d", version.Model, version.Firmware, version.Protocol),
```

Comment:
- Don't we need a separate `model` and `protocol`?
- Why is firmware in the metadata twice now?

Reply:
> Don't we need a separate model and protocol?

Yeah but the `protocol` field is not used anywhere. If there's a need to export these metrics as part of events for example, I'll update it here based on what's needed.

> Why is firmware in the metadata twice now?

None of these fields are actually recognised by the NOC currently. It only knows the `platform` field. This is used [here](https://github.com/TheThingsIndustries/gatewaymapdata/pull/12) to extract the version.

Reply:
> None of these fields are actually recognised by the NOC currently. It only knows the `platform` field. This is used [here](https://github.com/TheThingsIndustries/gatewaymapdata/pull/12) to extract the version.

OK so we put a formatted string in here to wire through the V2 NOC, to regex it for the V2 gateway map. Please file a technical debt TODO issue and reference it here to remove this.

Reply:
[Done](https://github.com/TheThingsNetwork/lorawan-stack/issues/2629)

Reply:
Missing the reference here